arbitrator/src/arbitrator.cpp

```diff
@@ -128,6 +128,6 @@ namespace arbitrator
     {
         ROS_INFO_STREAM("Arbitrator shutting down...");
         ros::shutdown();
-        exit(0);
+        std::exit(0);
```

Comment:
Actually, I think if you get rid of this line directly, ros::shutdown() would stop the spin and the program will end because main function ended.

Reply:
Good point, I'll go ahead and remove it.